WorkspaceServer.Tests/Kernel/CSharpKernelTests.cs

```diff
@@ -464,5 +465,75 @@ public async Task OnLoadAsync(IKernel kernel)
                         .ContainSingle(e => e.Value is CodeSubmissionEvaluated &&
                                             e.Value.As<CodeSubmissionEvaluated>().Code.Contains("using System.Reflection;"));
         }
+
+        [Fact]
+        public async Task Loads_native_dependencies_from_nugets()
+        {
+            var kernel = new CompositeKernel
+            {
+                new CSharpKernel().UseNugetDirective(new NativeAssemblyLoadHelper())
+            };
+
+            var command = new SubmitCode(@"#r ""nuget:Microsoft.ML, 1.3.1""
```

Comment:
@eerhardt Can you suggest a minimal and fast code submission to verify that the native assembly was correctly loaded?

Reply:
If you want something light-weight, maybe not using `Microsoft.ML` and using a different package that has native assemblies?  One that I know of https://www.nuget.org/packages/LibGit2Sharp. That actually is a PERFECT test case, if you look at it's `runtimes` folders, it hits exactly the issue I was referring to above:

![image](https://user-images.githubusercontent.com/8291187/63979568-d1705200-ca7e-11e9-9394-e36fb310138e.png)

Reply:
If you want to continue using ML.NET, the below code is probably the fastest thing I can think of from a public API perspective. Clustering 4 rows of 4 floats is super fast.